vmdb/app/models/ems_microsoft/powershell.rb

```diff
@@ -53,6 +59,25 @@ def run_dos_command(command)
       end
 
       $scvmm_log.debug("#{log_header} Execute DOS command <#{command}>...Complete - Timings: #{timings}")
+
+      results
+    end
+
+    def run_powershell_script(script)
```

Comment:
This method and run_dos_command are different by one line of code.  If would be nice to refactor them into a single method.  Not required for this PR.